src/core/exec-invoke.c

```diff
@@ -4029,6 +4033,21 @@ static int send_handoff_timestamp(
         return 1;
 }
 
+struct sched_attr {
```

Comment:
And please move this into missing_sched.h

Reply:
Done, thanks for your review. Not sure if I did everything right.